src/index.html

```diff
@@ -4,13 +4,47 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="search-form"
+    >
+      <input
+        type="text"
+        name="searchBig"
+        class="input"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+      class="search-form search-form--sm"
+    >
+      <input
+        type="text"
+        name="searchSmall"
+        class="input input--sm"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
   </body>
```

Comment:
```suggestion
    </form>
  </body>
```
don't add spaces between child and parent elements